recipes/pdb2pqr/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "pdb2pqr" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a6f5e2eefbd8bd2a072a89ee4026977de02dadd56e921095afac88d596b63820
+
+build:
+  number: 0
```

Comment:
Any reason this can't be `noarch: python`? Looks like this package is pure python.

Reply:
That's a good point; this is pure Python.  @tsenapathi -- what do you think?

Reply:
I agree and changed to` noarch: python`